components/enterprise-contract/ecp.yaml

```diff
@@ -129,4 +129,4 @@ spec:
         - github.com/release-engineering/rhtap-ec-policy//data
       name: Default
       policy:
-        - oci::quay.io/enterprise-contract/ec-task-policy:git-2689d04@sha256:4119ae3cbf40efab48301db357fd509ccd3cc609d059050feaf54cfbbbb0664e
+        - oci::quay.io/enterprise-contract/ec-task-policy:git-2689d04@sha256:aab6d47217b59d98e949ea161f65b98980b56862dfa01fe82e6a221eb246df9e
```

Comment:
Looks like all the tags changed to `git-e3f6361` except this one, was that expected?

Reply:
Yeah, for some reason the tag moved to a different digest and this is now causing issues because the previous blob was garbage collected.

Reply:
What I meant is that all the other tags in this PR moved to a different `git-<shortcommit>` tag, only this one stayed the same

Reply:
That corresponds to the [latest tag](https://quay.io/repository/enterprise-contract/ec-task-policy?tab=tags&tag=latest), so as expected. The change of the digest and the blob replacement was not though...

Reply:
Ack